workflow/rocoto/gefs_tasks.py

```diff
@@ -483,3 +483,48 @@ def wavepostpnt(self):
         task = rocoto.create_task(member_metatask_dict)
 
         return task
+
+    def extractvars(self):
+        deps = []
+        if self.app_config.do_wave:
+            dep_dict = {'type': 'task', 'name': f'wave_post_grid_mem#member#'}
+            deps.append(rocoto.add_dependency(dep_dict))
+        if self.app_config.do_ocean:
+            dep_dict = {'type': 'metatask', 'name': f'ocean_prod'}
+            deps.append(rocoto.add_dependency(dep_dict))
+        if self.app_config.do_ice:
+            dep_dict = {'type': 'metatask', 'name': f'ice_prod'}
+            deps.append(rocoto.add_dependency(dep_dict))
+        if self.app_config.do_atm:
+            dep_dict = {'type': 'metatask', 'name': f'atmos_prod'}
```

Comment:
Should these dependencies also be based on the member number task (like the wave dependency) rather than the whole metatask?

Reply:
Yes, these dependencies should only be based on the member number task. Will update the dependencies.

Reply:
I have added these changes, and I am rerunning the entire workflow (using the CI GEFS test case) to make sure everything works as expected. After this is done, I'll push these changes and ask for a re-review.